cmake/Modules/MacroQbtCXXCompilerAndModeCheck.cmake

```diff
@@ -0,0 +1,20 @@
+macro(qbt_cxx_compiler_and_mode_check)
```

Comment:
I'd change this to function and pass the minimal standard value in argument

Reply:
I wanted to suggest something similar... It would be good to define the minimal standard in the main (top level) project file.

Reply:
@FranciscoPombal, do you intend to finish it?

Reply:
@glassez Yes, sorry for the delay. I should have it ready in the following days.

Reply:
@glassez @zeule fixed.